library/alloc/src/boxed.rs

```diff
@@ -1146,9 +1150,13 @@ impl<T: ?Sized, A: Allocator> Box<T, A> {
     #[inline]
     pub const fn leak<'a>(b: Self) -> &'a mut T
     where
-        A: 'a,
+        A: PinSafeAllocator + ~const PinSafeAllocator,
```

Comment:
Do you mind changing this to `A: ~const PinSafeAllocator` everywhere else too?

Reply:
Fixed